tests/unit-test/Sitecore.Ship.AspNet.Test/packages.config

```diff
@@ -3,7 +3,7 @@
   <package id="Castle.Core" version="3.3.3" targetFramework="net45" />
```

Comment:
do we need a dependency to Castle.Core 3.3.3.?

Reply:
I haven't made any change to the Castle.Core libraries as it was not required to change or update.

Reply:
👍